h/views/activity.py

```diff
@@ -92,31 +95,76 @@ def group_search(request):
 
     return result
 
+
+@view_config(route_name='activity.user_search',
+             request_method='GET',
+             renderer='h:templates/activity/search.html.jinja2')
+def user_search(request):
+    if not request.feature('search_page'):
+        raise httpexceptions.HTTPNotFound()
+
+    username = request.matchdict['username']
+
+    result = search(request)
+
+    result['opts'] = {'search_username': username}
+    result['more_info'] = 'more_info' in request.params
+
+    user = request.find_service(name='user').fetch(
+        'acct:{username}@{authority}'.format(username=username,
+                                             authority=request.auth_domain))
+
+    if not user:
+        return result
+
+    def domain(user):
+        if not user.uri:
+            return None
+        return urlparse.urlparse(user.uri).netloc
+
+    result['user'] = {
+        'name': user.display_name or user.username,
+        'num_annotations': result['total'],
+        'description': user.description,
+        'registered_date': user.registered_date.strftime('%B, %Y'),
+        'location': user.location,
+        'uri': user.uri,
+        'domain': domain(user),
+        'orcid': user.orcid,
```

Comment:
In the future we might want a `UserHTMLPresenter` class in `h.presenters` for all this